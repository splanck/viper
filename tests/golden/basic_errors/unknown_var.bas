10 LET A = 1
20 PRINT B
30 END
