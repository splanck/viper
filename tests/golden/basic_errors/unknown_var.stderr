B1001.*did you mean 'A'
